tap_oracle/__init__.py

```diff
@@ -77,26 +81,20 @@ def schema_for_column(c, pks_for_table):
    data_type = c.data_type.lower()
    result = Schema()
 
-   numeric_scale = c.numeric_scale or DEFAULT_NUMERIC_SCALE
+   numeric_scale = c.numeric_scale if c.numeric_scale is not None else DEFAULT_NUMERIC_SCALE
    numeric_precision = c.numeric_precision or DEFAULT_NUMERIC_PRECISION
```

Comment:
Do you need the same `if not None` logic here?

Reply:
No, but maybe a comment. Precision's `None` state actually does seem to be 38 as far as we can tell. Whereas, for `scale`, None is semantically significant to differentiate from "number" vs. "int".